code/modules/ruins/spaceruin_code/clownstation.dm

```diff
@@ -0,0 +1,56 @@
+
+/obj/item/paper/fluff/ruins/clownstation/genesis
+	name = "torn out Honkspel page"
+	info = "In the beginning, the Honkmother honked her holy horn, and in a burst of sonic joy, the universe honked into existence. The Honkmother honked her horn a second time, and out of it honked the essence of life. The Honkmother collected the essence together, and it formed the shape of a pie. The Honkmother saw that the universe was not joyful, so she reached down her throat into the bowels of her spirit and pulled out a mass of banana. She molded the soft banana into several humanoid shapes, and threw the pie of life at them. From this, the first Clowns were born. The Honkmother scattered the clowns throughout the universe. However, a holy honk has balance, and with the creation of the essence of life came the creation of the essence of death. This pure malice formed itself into sickening beings. They were pale like the Clowns, but possessed no color in their garb. They spread themselves throughout the universe in equal number to the clowns, and wherever they met, chaos ensued. Where there is evil, there is good. Where there is hatred, there is love. When the Clowns and mimes procreated, they formed the common men and women of the galaxy. The Clowns tried their best to spread joy, but the common men and women wee never pleased. The Honkmother felt bad for her children and made them a safe haven in the remote corners of the cosmos. It is said that a Clown who's pranks are pure and honkfull will transcend to this paradise. There, the bananas glow in golden radiance brighter than the sun. The sky honks with the jubilant tone of a honkfull and balanced universe. All that is good is bountiful, and all that is mimely is absent. There is also a hell for Clowns. It is vast and bleak, and full of wicked mimery. Clowns who lay down their masks are sent there. Come now, and partake in the divine prank of the Honkmother. May your pies be bountiful and your existence honkfull."

```

Comment:
> but the common men and women wee never pleased

were*

I pressed the wrong button in my last comment and cant delete it, this is the like that needs fixing

Reply:
thank you for taking the time to read that whole section. I'll make a not to change it

Reply:
*note